README.md

```diff
@@ -1,19 +1,15 @@
 tf_aws_nixos_ami
-=================
+================
 
 Terraform module to get the current set of publicly available nixos AMIs.
 
-This module grabs all of the AMIs listed at:
-
-    https://nixos.org/wiki/NixOS_on_Amazon_EC2
-
-and then looks up the one you want given the input variables
-
 ## Input variables
 
-  * region - E.g. eu-central-1
-  * version - E.g. 14.12
-  * type -  s3/hvm/ebs
+  * release - E.g. 16.04
+  * root-device-type - ebs | instance-store
```

Comment:
Update documentation for variable names. They are with underscore instead of hyphen.
`virtualization_type` and `root_device_type`